TWLight/resources/forms.py

```diff
@@ -34,3 +35,24 @@ def __init__(self, *args, **kwargs):
             # Translators: This labels a button which users click to submit their suggestion.
             Submit("submit", _("Submit"), css_class="twl-btn"),
         )
+
+
+class SuggestionMergeForm(forms.Form):
+
+    suggestions_to_merge = forms.ModelMultipleChoiceField(
+        queryset=Suggestion.objects.all()
+    )
+    suggestions_merged_into = forms.ModelChoiceField(queryset=Suggestion.objects.all())
+
+    def __init__(self, *args, **kwargs):
+        super(SuggestionMergeForm, self).__init__(*args, **kwargs)
+        self.fields["suggestions_to_merge"].label = _("Merged suggestions")
```

Comment:
You need to add a translator comment to this label

Reply:
Any changes to this @suecarmol ? 😅

Reply:
> @ParitoshKabra the CI is still failing due to a Black formatting error

I ran the formatter but still the CI shows as build failed, can you point out the reason for the same @suecarmol ?
Thanks for bothering !